### PR TITLE
@broskoski: Fork personalize flow for New Collector project ("Artsy Primer") onboarding

### PR DESCRIPTION
**Pull request by craigspaeth**
Sorry for the huge diff, it's mostly copy + pasting what's in [/personalize](https://github.com/artsy/force/tree/master/desktop/apps/personalize). Artsy Primer is an experimental project we're launching only to a set of specific users and as such it's a bundle of quick hacks from Marketo pages to slightly modified Force UIs like this—so not looking for a serious code review here, just putting it on someone's radar.

This forks the code from our /personalize app into the /artsy_primer app to be able to send people through a slightly modified flow for the sake of keeping them in the "Artsy Primer" experience. This is a quick v1 version—there is a more dramatic v2 onboarding redesign which we should definitely rewrite with React if we do b/c there's a lot of state/render-syncing that can be made much simpler with better tools that I noticed as I dusted this code off.

cap

Noticed TODO from screencap:

- [ ] Open signup mode b/c users will likely not have an account
- [ ] Prefill inputs with query params (coming from email)
- [ ] Scroll to top on step change
- [ ] UI tweaks to price range selector

### CODE REVIEW

File: desktop/apps/artsy_primer/personalize/client/views/favorites.coffee [hunk 1]
```diff
@@ -28,12 +25,18 @@ module.exports = class FavoritesView extends StepView
...
+        @onFavorited()
 
   setupFavorites: ->
     @user.initializeDefaultArtworkCollection()
     @favorites = @user.defaultArtworkCollection()
     @favoriteArtworks = @favorites.get 'artworks'
-    @listenTo @favoriteArtworks, 'add remove', @setSkipLabel, this
+    @listenTo @favoriteArtworks, 'add remove', @onFavorited, this
+
+  onFavorited: =>
```
Comment (broskoski) [inline desktop/apps/artsy_primer/personalize/client/views/favorites.coffee]: 👍

File: desktop/apps/artsy_primer/personalize/client/views/next_step.coffee [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    )
+    if @state.get('current_step') is 'favorites' and @favorites?.length < 5
+      @$('.artsy-primer-next-step-button').attr('disabled', true)
+    else
+      @$('.artsy-primer-next-step-button').removeAttr('disabled')
+    if @favorites?.length and @state.get('current_step') is 'favorites'
+      @$('.artsy-primer-next-step-artworks').html template artworks: @favorites
+    else
+      @$('.artsy-primer-next-step-artworks').html ''
+    this
```
Comment (broskoski) [inline desktop/apps/artsy_primer/personalize/client/views/next_step.coffee]: Oof, yeah. This is a wild `render`

File: desktop/components/commercial_filter/filters/price/price_filter_view.coffee [hunk 3]
```diff
@@ -46,7 +46,7 @@ module.exports = class PriceFilterView extends Backbone.View
           "#{formatMoney(val, { precision: 0 })}"
         else
           "#{formatMoney(val, { symbol: "", precision: 0 })}"
-
+    debugger
```
Comment (broskoski) [inline desktop/components/commercial_filter/filters/price/price_filter_view.coffee]: Debugger here

File: desktop/components/slider/README.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+# Slider
```
Comment (broskoski) [inline desktop/components/slider/README.md]: Ah thanks for this!

**Comment by craigspaeth** [comment]
Thanks will fix up and re-PR

**Pull request closed without merging**